app/javascript/stylesheets/work.scss

```diff
@@ -58,3 +58,11 @@
     margin-left: 10px;
   }
 }
+
+.work-editor {
+  .version {
+    background-color: $stanford-digital-pink;
+    border-color: $stanford-cardinal;
+    border-width: 3px;
+  }
```

Comment:
I believe you should be able to move these lines into `app/javascript/stylesheets/workEditor.scss` which already has a `.work-editor` block in it.